### PR TITLE
Feature Network URL

### PR DESCRIPTION
**Pull request by ihorml**
**Description**

When you switch to `testnet` any page change in the app will lead to `?network` parameter in the URL. If you switch back to `mainnet` - the `?network` will be removed.

### CODE REVIEW

**Comment by ihorml** [comment]
Screenshot 2022-12-23 at 14 26 05
just noticed there is a different title and the changes by network dropdown doesn't affect it without page reload. Will fix it

**Comment by ihorml** [comment]
done & pushed

File: app/core/config.route.js [hunk 1]
```diff
@@ -23,50 +23,67 @@
             routes.forEach(function(route) {
                 return setRoutes(route);
             });
+            
+            const redirectTo = (params,url) => {
+                // the idea of this script is to keep always "?network={network}" in user's url
+                // if he surfing on a testnet blockchain
+                const activeBlockchain = ApiProvider.getActiveBlockchain();
+                
+                if(url.indexOf('?network') === -1 && activeBlockchain.key === 'testnet') {
```
Comment (sschiessl-bcp) [inline app/core/config.route.js:32]: The premise is not actually "if testnet", the premise is "if not default"

**Review by sschiessl-bcp — CHANGES_REQUESTED** [review]

File: app/core/config.route.js (hunk 1 above)
Comment (ihorml) [reply app/core/config.route.js:32]: Okay, let me clarify first. Are we talking about the `network` or `blockchain`?

**Comment by ihorml** [comment]
@sschiessl-bcp pushed!

**Comment by sschiessl-bcp** [comment]
replaced

**Pull request closed without merging**